src/linux/cgroups2.cpp

```diff
@@ -786,6 +787,23 @@ Try<Bytes> usage(const string& cgroup)
   return Bytes(*contents);
 }
 
+
+Try<Nothing> set_min(const string& cgroup, const Bytes& bytes)
+{
+  return cgroups2::write(cgroup, control::MIN, stringify(bytes.bytes()));
+}
+
+
+Try<Bytes> min(const string& cgroup)
+{
+  Try<string> contents = cgroups2::read<string>(cgroup, control::MIN);
+  if (contents.isError()) {
+    return Error("Failed to read 'memory.min': " + contents.error());
+  }
+
+  return Bytes::parse(strings::trim(*contents) + "B");
```

Comment:
no need to parse with a B here, can just cgroups::read a uint64_t and directly construct Bytes with it?